tests/test_pif.py

```diff
@@ -92,7 +102,11 @@ def test_PWSCF(self):
             
             # Make the pif file
             # print("\tpif for example:", name)
-            result = directory_to_pif(name)
+            try:
```

Comment:
Was this debugging code?

Reply:
Yes. Does it make sense to keep it in here for the unit tests?

Reply:
Yeah; I guess its useful.